demoRedBall/src/main.cpp

```diff
@@ -824,7 +825,7 @@ class managerThread : public PeriodicThread
             }
         }
         else if (((state==STATE_IDLE) || (state==STATE_REACH)) &&
-                 ((Time::now()-idleTimer)>idleTmo) && !wentHome)
+                 ((Time::now()-idleTimer)>idleTmo) && !wentHome && !simulation)
```

Comment:
This addition is because, when we're in `simulation` and we ask the robot to look down at startup, the robot needs to keep looking down instead of going home.